### ISSUE TITLE
List of page numbers below program grid can partially obscure X-axis slider - 1.16.0

### ISSUE THREAD

**Issue by Mostelin** [christofmuc/KnobKraft-orm#172]
When there are many pages of programs, the list of pages numbers below the grid is intelligently telescoped down to fit the space available. For example, if a user is looking at page 17 of 57 pages the list will show as:

1 ...15 16 17 18 19 ... 57

However, if the user selects only categories of a certain size the list can expand in length up to:

1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16

which almost completely obscures the X-axis slider. Above 16 items, the list is telescoped properly.

I've also just noticed that with smaller window sizes the untelescoped list can actually obscure both sliders.

**Comment by christofmuc** [comment]
@Mostelin Good point, I was eyeing that area already. I need to see why I am that generous with page number space, it seems a bit excessive!
